cmd/gapit/flags.go

```diff
@@ -409,4 +409,13 @@ type (
 		DeviceFlags
 		Gapis GapisFlags
 	}
+
+	PerfettoFlags struct {
+		Mode				string	`help:"Run mode: {metrics|interactive}. Default: metrics."`
```

Comment:
Instead of a string you can use an enum here.

See core/log/severity.go for how this is done for (--log-level "Debug|Verbose|....)

Reply:
Done.